desktop/core/src/desktop/settings.py

```diff
@@ -336,6 +336,8 @@
     ],
     'DEFAULT_AUTHENTICATION_CLASSES': desktop.conf.AUTH.API_AUTH.get()
 }
+if desktop.conf.AUTH.JWT.IS_ENABLED.get() and 'desktop.auth.api_authentications.JwtAuthentication' not in REST_FRAMEWORK['DEFAULT_AUTHENTICATION_CLASSES']:
```

Comment:
Making sure we dont add the duplicate and lose the priority which user sets in the config.

We can have another check which can rectify the users mistake which is as follows:

```
if 'desktop.auth.api_authentications.JwtAuthentication' in REST_FRAMEWORK['DEFAULT_AUTHENTICATION_CLASSES'] and not desktop.conf.AUTH.JWT.IS_ENABLED.get():
...remove the auth class?
```
@romainr

Reply:
I believe it is good/KISS enough how you did it!

desktop.conf.AUTH.API_AUTH.get() should be the ultimate source of truth (i.e. overrides everything)